packages/cdktf-cli/src/bin/cdktf.ts

```diff
@@ -111,6 +111,10 @@ yargs
     required: false,
     desc: "Which log level should be written. Only supported via setting the env CDKTF_LOG_LEVEL",
   })
+  .option("log-file-directory", {
```

Comment:
As we don't pass forward this flag, we might need to add a disclaimer as in the flag above:
> Only supported via setting the env CDKTF_LOG_LEVEL

Reply:
Good catch! Updated.